### PR TITLE
Genivi imp

### PR DESCRIPTION
**Pull request by GeniviAMmaintainer**
follow up to PR#36 closed by mistake

### CODE REVIEW

**Comment by GeniviAMmaintainer** [comment]
merged and closed the last PR#36 by mistake. Reverted master branch and created new PR for following up. Database macros cannot be used as suggested, other changes were applied.

File: AudioManagerCore/src/CAmDatabaseHandlerMap.cpp [hunk 1]
```diff
@@ -1492,35 +1501,39 @@ am_Error_e CAmDatabaseHandlerMap::changeMainSinkSoundPropertyDB(const am_MainSou
         logError(__METHOD_NAME__, "sinkID must exist");
         return (E_NON_EXISTENT);
     }
-    bool modified = false;
+
+    bool                                          modified        = false;
```
[the remark this answers is not in the document]
Comment (GeniviAMmaintainer) [reply AudioManagerCore/src/CAmDatabaseHandlerMap.cpp]: We checked and they don't fit exactly. And it seems they work differently depending on the make configuration.

[the remark this answers is not in the document]
Comment (JensLorenz) [reply AudioManagerCore/src/CAmDatabaseHandlerMap.cpp]: This is right. I thought this is the intention. Either you are interested to filter messages or you are interested to be modified in all notification even there is no change.
This is the switch about.

Background: The un-filtered situation is the old implementation and there might be applications which relay on this behaviour. If you have a good design this is not needed and you can activate the switch.

Expectation:
I request to implement according to the macros.
We can decide later to remove the switch and align the entire code to a common solution.

**Comment by JensLorenz** [comment]
Looks good to me. Would recommend to squash the patches together and rebase the PR in order to keep to patch files.

**Comment by JensLorenz** [comment]
Actually now you rebased and squashed everything together in one single patch.
I was requesting you to have only the CAmDatabaseHandler modifications to merged into one patch.
DLT should still seperate.

**Comment by GeniviAMmaintainer** [comment]
Anyway, I think it is still ok to keep the squashed commit - there are not so many files changed.

**Pull request merged**